pkg/istio-agent/sds-agent.go

```diff
@@ -199,7 +199,7 @@ func NewSDSAgent(discAddr string, tlsRequired bool) *SDSAgent {
 		ac.RequireCerts = true
 		// For local debugging - the discoveryAddress is set to localhost, but the cert issued for normal SA.
 		if discHost == "localhost" {
-			discHost = "istiod.istio-system"
+			discHost = "istiod.istio-system.svc"
```

Comment:
this change because istiod's dns cert now is "istoid.istio-system.svc"?

Reply:
yes - consistency with kube conventions for webhook, no need to get 2 different certs

Reply:
I would suggest we support both, as currently the pilot cert dns includes both `istio-pilot.istio-system` and `istio-pilot.istio-system.svc`